draft-frost-rats-eat-collection.md

```diff
@@ -183,8 +183,8 @@ The media type introduced here builds on top and extends the EAT media types {{-
   of {{&SELF}}
 * Interoperability considerations: n/a
 * Published specification: {{&SELF}}
-* Applications that use this media type: Attesters and Relying Parties sending
-  EAT collection attestation message over HTTP(S) and other transports.
+* Applications that use this media type: Attesters, Relying Parties and Verifiers sending
+  EAT collection attestation messages over HTTP(S) and other transports.
```

Comment:
quibbling: "attestation" is made redundant by the very clear and specialised context.

Reply:
Yes, I spotted it yesterday, but got to rush home so did not made the commit. Will do it now!